Modules/Sources/Networking/Remote/POSCatalogSyncRemote.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+
+/// POS Catalog Sync: Remote Endpoints
+///
+public class POSCatalogSyncRemote: Remote {
+    /// Loads POS products modified after the specified date.
+    ///
+    /// - Parameters:
+    ///   - modifiedAfter: Only products modified after this date will be returned.
+    ///   - siteID: Site ID to load products from.
+    ///   - productTypes: Product types to filter by.
+    ///   - pageNumber: Page number for pagination.
+    /// - Returns: Paginated list of POS products.
+    ///
+    // periphery:ignore - TODO - remove when this endpoint is integrated with catalog sync
+    public func loadProducts(modifiedAfter: Date, siteID: Int64, productTypes: [ProductType] = [.simple, .variable], pageNumber: Int)
+    async throws -> PagedItems<POSProduct> {
+        let path = "products"
+        let dateFormatter = ISO8601DateFormatter()
```

Comment:
```suggestion
        let dateFormatter = ISO8601DateFormatter()
```
Perhaps best to have this in a property? I think `DateFormatter` is reasonably expensive to create, and this could be called quite a few times.

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/16033/commits/b1f985064aadba165373fc77d09e91ea4f3b7875.